### ISSUE TITLE
Internal pip failure with pip==20.0.2

### ISSUE THREAD

**Issue by nedbat** [signalpillar/tox-battery#19]
I get this error on Appveyor on WIndows:
```
Traceback (most recent call last):
  File "c:\python36-x64\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36-x64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36-x64\Scripts\tox.exe\__main__.py", line 7, in <module>
  File "c:\python36-x64\lib\site-packages\tox\session\__init__.py", line 44, in cmdline
    main(args)
  File "c:\python36-x64\lib\site-packages\tox\session\__init__.py", line 64, in main
    config = load_config(args)
  File "c:\python36-x64\lib\site-packages\tox\session\__init__.py", line 80, in load_config
    config = parseconfig(args)
  File "c:\python36-x64\lib\site-packages\tox\config\__init__.py", line 252, in parseconfig
    pm = get_plugin_manager(plugins)
  File "c:\python36-x64\lib\site-packages\tox\config\__init__.py", line 71, in get_plugin_manager
    pm.load_setuptools_entrypoints("tox")
  File "c:\python36-x64\lib\site-packages\pluggy\manager.py", line 299, in load_setuptools_entrypoints
    plugin = ep.load()
  File "c:\python36-x64\lib\site-packages\importlib_metadata\__init__.py", line 94, in load
    module = import_module(match.group('module'))
  File "c:\python36-x64\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\python36-x64\lib\site-packages\toxbat\requirements.py", line 44, in <module>
    from pip._internal.download import PipSession
ModuleNotFoundError: No module named 'pip._internal.download'
```
But it doesn't happen on Travis.  I'm not sure what the OS has to do with it, probably the ordering of installation?

**Comment by signalpillar** [comment]
Hi @nedbat , thanks for reporting this issue.

I guess it is due to latest version of pip that probably has a different API. Need to check it.

**Comment by signalpillar** [comment]
Hi @nedbat I published a patch version 0.5.2 it should fix the issue.
